app/helpers/application_helper/button/smart_state_scan.rb

```diff
@@ -0,0 +1,19 @@
+class ApplicationHelper::Button::SmartStateScan < ApplicationHelper::Button::Basic
+  needs :@record
+
+  SMART_ROLES = %w(SmartProxy SmartState).freeze
```

Comment:
The only place I could think of is in `MiqServer::ServerSmartProxy` but I am also not sure about it.

[1] https://github.com/ManageIQ/manageiq/blob/master/app/models/miq_server/server_smart_proxy.rb#L3